pkg/k8s/identitybackend/identity.go

```diff
@@ -228,6 +230,20 @@ func (c *crdBackend) get(ctx context.Context, key allocator.AllocatorKey) *v2.Ci
 		return nil
 	}
 
```

Comment:
I'd suggest adding comment to tell why it is sorted here.

Reply:
I have updated the function comment, is that not enough? I mean "duplicates, therefore we are selecting the 'lexicographically smalles' name" implies to certain extent we will 'sort' the results before deciding which one to return.